templates/error/404.jsx

```diff
@@ -3,25 +3,31 @@ import { Button } from '@oacore/design'
 
 import styles from './404.module.css'
 
-const Error404 = ({ articleId, status }) => (
-  <article className={styles.errorContainer}>
-    <section className={styles.error}>
-      <h1>Uh-oh</h1>
-      {articleId && status === 410 ? (
-        <p>The article with ID {articleId} has been disabled.</p>
```

Comment:
I don't think this should be removed... The outputs will return 410 when the article will have status=deleted...